release-1.8/release_notes_draft.md

```diff
@@ -39,9 +39,9 @@ please check out the [release notes guidance][] issue.
 
 - The kubernetes workloads API (the DaemonSet, Deployment, ReplicaSet, and
 StatefulSet kinds) have been moved to the new apps/v1beta2 group version. This
-is the current version of the API, and the version we intend to promote to 
-GA in future releases. This version of the API introduces several deprecations 
-and behavioral changes, but its intention is to provide a stable, consistent 
+is the current version of the API, and the version we intend to promote to
```

Comment:
Is it necessary to state that the API will be promoted to GA in a future release?

Reply:
This is not my part, it was just reformatted during my changes. Just removed newlines. So I'm not going to change this part.